tools/pythonpkg/tests/fast/test_relation.py

```diff
@@ -71,6 +71,16 @@ def test_projection_operator_double(self):
         rel = get_relation(conn)
         assert rel.order('j').execute().fetchall() == [(4, 'four'), (1, 'one'), (3, 'three'), (2, 'two')]
 
+    def test_projection_mixed_types(self):
```

Comment:
This test does not stress the new functionality added at all, since your string is just `'i'` this was already covered by the old behavior, this would implicitly become a `ColumnExpression`